src/main/navigation/navigation.c

```diff
@@ -41,8 +41,8 @@
 #include "fc/rc_modes.h"
 #include "fc/runtime_config.h"
 #ifdef USE_MULTI_MISSION
-#include "fc/rc_adjustments.h"
-#include "fc/cli.h"
+#  include "fc/rc_adjustments.h"
```

Comment:
silly, but this whitespace is strange :)

Reply:
There is nothing about it. I have seen those whitespaces elsewhere in the INAV code and tried to keep the look consistend... unless I encountered random usage/not-usage of whitspaces with includes. xD

Reply:
Whitespaces sent to hell.